Automattic/pocket-casts-ios
# Single Sign On

| 📘 Project: #381 |
|:---:|

* Adds sign-in with Google and Apple
* This is still hidden behind a feature flag

## To test

In order to merge we just need to make sure that:

1. Currently login works just fine for new logins
2. User migrating from an older version to this code doesn't see any changes
3. Token refresh works as intended

So:

1. Run `trunk` (without this branch code)
2. Login
3. Checkout this branch and run the app
5. Go to Profile > tap "Refresh Now"
6. ✅ The sync should succeed
7. Go to Profile > Settings > Developer > Corrupt Sync Login Token
4. Go to Profile > tap "Refresh Now"
5. ✅ The sync should succeed
5. Log out
5. Login again
5. ✅ You should login into your account

## Checklist
